dm/ctl/master/start_task.go

```diff
@@ -29,34 +30,33 @@ func NewStartTaskCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "start-task [-s source ...] [--remove-meta] <config-file>",
 		Short: "start a task as defined in the config file",
-		Run:   startTaskFunc,
+		RunE:  startTaskFunc,
 	}
 	cmd.Flags().BoolP("remove-meta", "", false, "whether to remove task's meta data")
 	return cmd
 }
 
 // startTaskFunc does start task request
-func startTaskFunc(cmd *cobra.Command, _ []string) {
+func startTaskFunc(cmd *cobra.Command, _ []string) (err error) {
 	if len(cmd.Flags().Args()) != 1 {
 		cmd.SetOut(os.Stdout)
 		cmd.Usage()
+		err = errors.New("please check output to see error")
 		return
 	}
 	content, err := common.GetFileContent(cmd.Flags().Arg(0))
 	if err != nil {
-		common.PrintLines("get file content error:\n%v", err)
 		return
 	}
 
 	sources, err := common.GetSourceArgs(cmd)
 	if err != nil {
-		common.PrintLines("%v", err)
 		return
 	}
 
 	removeMeta, err := cmd.Flags().GetBool("remove-meta")
 	if err != nil {
-		common.PrintLines("%v", err)
+		common.PrintLines("error in parse remove-meta")
```

Comment:
a little suggestion, for `error in parse` error, how about showing the flag as `--remove-meta` or ``` `remove-meta` ```?

Reply:
https://github.com/pingcap/dm/pull/845/commits/425157f3d92faa1a42bda39a9e62edc52e8b694b